src/main/java/mate/academy/spring/dto/UserResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.dto;
+
+public class UserResponseDto {
+    private Long id;
+    private String name;
+    private String lastName;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
```

Comment:
mb better add toSring, equals and hashcode ?

Reply:
@sSlonIk it is not obligatory for dtos)